dev.yml

```diff
@@ -1,13 +1,13 @@
 ---
 name: kubernetes-deploy
 up:
-  - ruby: 2.3.3
```

Comment:
The reason for this version so that we're developing with the minimum supported in our gemspec: `spec.required_ruby_version = '>= 2.3.0'`. Are you saying minikube somehow depends on the ruby version? 😕

Reply:
TIL . I'll revert the ruby change, just seemed odd that we were using such an old version of ruby.

eq -> ge change is totally separate.

Reply:
Reasonable confusion. 😄 Can you add a comment?